docs/sources/panels/transformations/debug.md

```diff
@@ -0,0 +1,16 @@
++++
+title = "Debug Transformations"
+type = "docs"
+[menu.docs]
+identifier = "debug-transformations"
+parent = "tranformations"
+weight = 300
++++
+
+## Debug transformations
```

Comment:
```suggestion
# Debug transformations
```

Reply:
Every Hugo topic should start with an h1.

That said, this topic is short enough, you might fold it into Apply a transformation or Apply transformations.

Reply:
Wherever it ends up, please be sure to link it from the Troubleshooting section.

Reply:
Moved the debug section back to the Apply Transformations topic.